ReactWindows/ChakraBridge/JsIndexedModulesUnbundle.cpp

```diff
@@ -0,0 +1,121 @@
+#include "pch.h"
+#include "ChakraHost.h"
+#include "JsIndexedModulesUnbundle.h"
+
+JsIndexedModulesUnbundle::JsIndexedModulesUnbundle(const wchar_t* sourcePath)
+{
+    this->sourcePath = sourcePath;
+}
+
+JsIndexedModulesUnbundle::~JsIndexedModulesUnbundle()
+{
+	if (fileHandle != NULL)
+	{
+		UnmapViewOfFile(byteBuffer);
+		CloseHandle(mapHandle);
```

Comment:
Just following the lead of [https://github.com/ReactWindows/react-native-windows/blob/master/ReactWindows/ChakraBridge/SerializedSourceContext.h#L14-L24](https://github.com/ReactWindows/react-native-windows/blob/master/ReactWindows/ChakraBridge/SerializedSourceContext.h#L14-L24)

---
In reply to: [103548082](https://github.com/ReactWindows/react-native-windows/pull/1017#discussion_r103548082) [](ancestors = 103548082)